README.md

```diff
@@ -1,3 +1,83 @@
 # HyP3 Event Monitoring
 
-A software stack to allow users to enable event monitoring and processing of data as it becomes available.
\ No newline at end of file
+A software stack that allows automatic submission of jobs to hyp3 over a specified area to easily monitor areas of interest.
+
+## Table of contents
+- [Deployment](#deployment)
+  - [Prerequisites](#prerequisites)
+  - [Stack Parameters](#stack-parameters)
+  - [Deploy with CloudFormation](#deploy-with-cloudformation)
+- [Testing](#testing)
+
+## Deployment
+
+### Prerequisites
+These resources are required for a successful deployment, but managed separately:
+
+- HyP3 target deplotment
+- S3 bucket for CloudFormation deployment artifacts
+- EarthData Login account authorized to download data from ASF
+  - If submitting more jobs that the defualt hyp3 quota allows you may need to have an incresed quota
+- IAM user and roles for automated CloudFormation deployments (if desired)
+
+### Stack Parameters
+Review the parameters in [cloudformation.yml](cloudformation.yml) for deploy time configuration options.
+
+### Deploy with CloudFormation
+
+- Install dependencies for build and run
+```sh
+pip install -r requirements-all.txt
+```
```

Comment:
`requirements-all.txt` isn't necessary for a cloudformation deployment, can strike this section